tests/ui/typeck/issue-86721-return-expr-ice.rev1.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0572]: return statement outside of function body
+error[E0572]: `return` statement outside of function body
```

Comment:
I think I prefer this without the backticks...

Reply:
reverted the backticks